ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -57,6 +57,7 @@
 public class XARecoveryModuleUnitTest
 {
     private boolean rolledback;
+    private int recoverCount = 0;
```

Comment:
I'm also doubting that recoverCount needs to be declared as an instance variable: the asserts for the value don't add anything since recoverCount is internal to how the test operates, the code comments about recoverCount should be sufficient.

Reply:
Agreed, so I will restore the previous variable (count) and just rename it to 'recoverCount'.